src/refract/PrintVisitor.cc

```diff
@@ -6,180 +6,179 @@
 //  Copyright (c) 2015 Apiary Inc. All rights reserved.
 //
 
-#include "VisitorUtils.h"
-#include <sstream>
+#include <cassert>
 #include <iostream>
+#include "VisitorUtils.h"
 
 #include "PrintVisitor.h"
 #include "Visitor.h"
 
-#define VISIT_IMPL( ELEMENT ) void PrintVisitor::operator()(const ELEMENT ## Element& e) \
-    {                                                                    \
-        typedef ELEMENT ## Element::ValueType::const_iterator iterator;  \
-        PrintVisitor ps(indent + 1, os);                                 \
-        Visitor visitor(ps);                                             \
-                                                                         \
-        os << "" #ELEMENT "Element  {\n";                                \
-        for (iterator it = e.value.begin(); it != e.value.end(); ++it) { \
-            visitor.visit(*(*it));                                       \
-        }                                                                \
-        indentOS(indent);                                                \
-        os << "}\n";                                                     \
-    }                                                                    \
-
 namespace refract
 {
-    PrintVisitor::PrintVisitor()
-    : indent(0), os(std::cout) {}
+    PrintVisitor::PrintVisitor() : indent(0), os(std::cerr), ommitSrcMap(false)
```

Comment:
Please rename `ommitSrcMap` to `ommitSourceMap`. We need to use the full name of the variable.